salaTest/testlinkutils.cpp

```diff
@@ -97,15 +97,17 @@ TEST_CASE("Test linking - fully filled grid (no geometry)", "")
     {
         Point2f start(bottomLeft.x - spacing, bottomLeft.y - spacing);
         mergeLines.push_back(Line(start,topRight));
-        REQUIRE_THROWS_WITH(depthmapX::getLinksFromMergeLines(mergeLines,pointMap),
+        std::vector<PixelRefPair> links = depthmapX::pixelateMergeLines(mergeLines,pointMap);
+        REQUIRE_THROWS_WITH(depthmapX::mergePixelPairs(links, pointMap),
```

Comment:
These tests don't really belong here anymore, do they? They don't try the link parser, they test the link helpers and should be in there.